tools/accuracy_checker/accuracy_checker/annotation_converters/criteo_kaggle_dac.py

```diff
@@ -104,14 +107,24 @@ def convert(self, check_content=False, **kwargs):
 
             c_file = str(c_input.relative_to(preprocessed_folder))
 
-            for j in range(i, i + self.batch):
+            if self.block:
                 annotations.append(ClassificationAnnotation(
                     [
-                        "input.1_{}{}{}".format(j, self.separator, c_file),
-                        "lS_i_{}{}{}".format(j, self.separator, c_file),
-                        "lS_o_{}{}{}".format(j, self.separator, c_file),
+                        "input.1_{}{}{}".format(i, self.separator, c_file),
+                        "lS_i_{}{}{}".format(i, self.separator, c_file),
+                        "lS_o_{}{}{}".format(i, self.separator, c_file),
                     ],
-                    y[j, ...]
+                    y[i:i+self.batch, ...]
                 ))
+            else:
+                for j in range(i, i + self.batch):
+                    annotations.append(ClassificationAnnotation(
+                        [
+                            "input.1_{}{}{}".format(j, self.separator, c_file),
```

Comment:
I right understand it is model specific input name? Or is it common?

Reply:
Yes, model specific name. Are you want to map this in converter parameters?

Reply:
on my view better keep it neutral in the identifier and map in config using value for specific input layer.
Who knows, maybe in future we will get new model with different input name

Reply:
Ok

Reply:
Fixed